habitat-lab/habitat/tasks/rearrange/sub_tasks/cat_nav_to_obj_task.py

```diff
@@ -49,9 +68,16 @@ def _cache_receptacles(self):
             obj = rom.get_object_by_handle(obj_handle)
             user_attr_keys = obj.user_attributes.get_subconfig_keys()
             if any(key.startswith("receptacle_") for key in user_attr_keys):
-                self._receptacle_semantic_ids[
-                    obj.object_id
-                ] = obj.creation_attributes.semantic_id
+                obj_name = osp.basename(obj.creation_attributes.handle).split(
```

Comment:
Why not `obj.handle`?What does `obj.handle` give?

Reply:
We can get it from `obj.handle` as well, but I just felt this is cleaner since this gives us the file name, but we don't know what `obj.handle` could give us in the future.